shared/util/emoji.tsx

```diff
@@ -12,20 +13,13 @@ export type EmojiData = {
   source?: string
 }
 export function RPCToEmojiData(emoji: RPCChatTypes.Emoji, category?: string): EmojiData {
-  return emoji.source.typ === RPCChatTypes.EmojiLoadSourceTyp.str
-    ? {
-        category: category ?? '',
-        name: null,
-        short_name: emoji.source.str,
-        short_names: [emoji.source.str, emoji.alias],
-        unified: '',
-      }
-    : {
-        category: category ?? '',
-        name: null,
-        short_name: emoji.alias,
-        short_names: [emoji.alias],
-        source: emoji.source.httpsrv,
-        unified: '',
-      }
+  return {
```

Comment:
picker and autocomplete don't handle aliases well but that's probably another ticket

Reply:
I think with this change alias now works in the picker. I'll look into autocomplete